pom.xml

```diff
@@ -98,6 +98,11 @@
             <version>${storm.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Why do we need this?

Reply:
storm-core-2.1.0 has jackson version 2.9.8 but bulletavro uses avro 1.9.2 which uses jackson 2.10.2. We get noclassdeffounderrors without including it.

Reply:
Let's put this in dependencyManagement then since we don't actually need it.